userguide/sharing.rst

```diff
@@ -1299,14 +1300,15 @@ for more details.
    sure to configure **all** SMB shares this way. Reboot the Mac
    client after making this change.
 
-These VFS objects do not appear in the drop-down menu and are dynamically
-configured:
+These VFS objects do not appear in the selection box:
 
 * **recycle:** moves deleted files to the recycle directory instead of
-  deleting them. Controlled by a checkbox in the share configuration.
+  deleting them. Controlled by :guilabel:`Export Recycle Bin`.
 
 * **shadow_copy2:** a more recent implementation of
-  :guilabel:`shadow_copy` with some additional features
+  :guilabel:`shadow_copy` with some additional features. *shadow_copy2*
+  and associated *shadow:* parameters are automatically added to the
```

Comment:
Some VFS modules in samba take additional parameters as defined in the module. FreeNAS sets the following shadow_copy2 parameters:

- shadow:snapdirseverywhere = yes
- shadow:format = <snapshot task>
- shadow:localtime = yes
- shadow:sort = desc
- shadow:snapdir = .zfs/snapshot

I tried to just write "shadow:*" but wasn't sure how the markup language would like *shadow:**